company/about-mattermost/mindsets.md

```diff
@@ -20,37 +20,41 @@ In this context, here are mindsets for Mattermost:
 
 The best teachers can speed colleagues to surpass them. This mindset helps us constantly grow and rotate into new roles, while preventing single-points of failure where only one person is qualified for a certain task.
 
-### Slow is smooth, smooth is fast
+### Slow is Smooth, Smooth is Fast
```

Comment:
Why are these subheadings in title case?

Reply:
The rest of the headings are in title case so I wanted to make it consistent here. It's not in line with what's in the style guide but I'm not sure how long it'll take me to change all the headers here - so leaving it as-is for now until I can make a wider change.